src/App.vue

```diff
@@ -10,24 +10,19 @@ THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
       :class="`tool_${tool.toLowerCase()}`">
         <toolbar :class="{ 'disabled-component': tool === 'Map' }"></toolbar>
 
-        <resize-group :vertical="true">
-            <resize-group :horizontal="true" id="layout-main">
-                <resize id="layout-navigation" :resize-right="true" :resize-right-min="200">
-                    <navigation :class="{ 'disabled-component': tool === 'Map' }"></navigation>
-                </resize>
-                <main>
-                  <div id="alert-text" v-show="error || success" :class="{ error, success }">
-                      <p>{{ error || success }}</p>
-                  </div>
-                    <canvas-view></canvas-view>
-                    <grid-view></grid-view>
-                </main>
-                <!-- <inspector-view></inspector-view> -->
-            </resize-group>
-            <resize id="layout-library" :resize-top="true" :resize-top-min="100">
-                <library :class="{ 'disabled-component': tool === 'Map' }"></library>
-            </resize>
-        </resize-group>
+        <div id="layout-main">
+            <navigation :class="{ 'disabled-component': tool === 'Map' }"></navigation>
+
+            <main>
+              <div id="alert-text" v-show="error || success" :class="{ error, success }">
```

Comment:
this `:class="{ error, success }"` thing is super clever. I like it alot.
![](http://4.bp.blogspot.com/_D_Z-D2tzi14/S8TRIo4br3I/AAAAAAAACv4/Zh7_GcMlRKo/s400/ALOT.png)